.travis.yml

```diff
@@ -0,0 +1,38 @@
+language: python
+python:
+  - "2.7"
+before_install:
+  - sudo apt-get update -qq
+  - sudo apt-get install mpi-default-bin mpi-default-dev
+
+  # LAMMPS
+  #  - build lammps as shared library
+  #  - and install LAMMPS python wrapper
+  - git clone git://git.lammps.org/lammps-ro.git mylammps
+  # checkout a recent stable version (from 28 June 2014)
+  - cd mylammps/src && git checkout r12164
+  # make shared library for python
+  - make makeshlib && make -f Makefile.shlib ubuntu_simple
+  - cd ../python && sudo python install.py /usr/lib/ $VIRTUAL_ENV/lib/python2.7/site-packages/
+  # make lammps executable and add to path
+  - cd ../src
+  - make ubuntu_simple
+  - ln -s lmp_ubuntu_simple lammps
+  - export PATH=$TRAVIS_BUILD_DIR/mylammps/src/:$PATH
+  - cd $TRAVIS_BUILD_DIR
+
+  # use current master of simphony
+  - pip install git+git://github.com/simphony/simphony-common.git#egg=simphony
```

Comment:
> However, some of the CUBA keywords can be used also for the configuration (e.g, VELOCITY in BC for a fixed velocity boundary condition, and also for the particle velocity, etc).

While the CUBA keywords are one group creating subgroups does not mean that they cannot have overlapping  keywords. In fact we can implement the changes in such a way that the user is not even aware of the subgroups which will only used internally by the two subtypes of the DataContainer. The user will still use `CUBA.VELOCITY` in both DataContainers.